xcommon.cmake

```diff
@@ -481,6 +479,16 @@ function(XMOS_REGISTER_APP)
 
     set(LIB_DEPENDENT_MODULES ${APP_DEPENDENT_MODULES})
 
+    if(DEFINED XMOS_DEPS_ROOT_DIR)
+        message(WARNING "XMOS_DEPS_ROOT_DIR is deprecated; please use XMOS_SANDBOX_DIR instead")
+    endif()
+    if(NOT DEFINED XMOS_SANDBOX_DIR)
```

Comment:
Should this not be nested in the previous if?

Reply:
Logically I think the behaviour is the same in terms of strings being defined or empty, but I agree that it would make more sense to put this inside the previous because it is related to deprecation of that variable.

Reply:
Done.